app/assets/stylesheets/_size.scss

```diff
@@ -0,0 +1,2 @@
+$nav_height: 50px;
```

Comment:
Name of variable `nav_height` should be written in all lowercase letters with hyphens instead of underscores